app/controllers/admin/articles_controller.rb

```diff
@@ -15,6 +15,7 @@ def new
     def create
       @article = Article.new(article_params)
       if @article.save
+        @article.touch(:created_at)
```

Comment:
Using `touch` here will cause a second call to the database. Also if that second call fails the redirect will continue without the `created_at` being updated.

I'd suggest updating line 16 to:
```ruby
@article = Article.new(article_params.merge(created_at: Time.now.utc))
```
Rather than adding the `touch` call. That way, there will only be one write call to the database, and the redirect will only occur if that is successful.